src/watchman_watcher.js

```diff
@@ -69,16 +69,18 @@ WatchmanWatcher.prototype.init = function() {
     return self.watchProjectInfo ? self.watchProjectInfo.root : self.root;
   }
 
-  function onVersion(error, resp) {
+  function onCapability(error, resp) {
     if (handleError(self, error)) {
+      // The Watchman watcher is unusable on this system, we cannot continue
+      process.exit(1);
```

Comment:
But this is nothing new. We did that before with the `which watchman` https://github.com/amasad/sane/pull/55/files#diff-d062b2ce5d28d903878be707d22d8b65L313

How do you propose to handle this? Emit an error?